src/crowdin_gui.cpp

```diff
@@ -385,20 +386,13 @@ class CrowdinOpenDialog : public wxDialog
     {
         m_info = prj;
         // Put supported files first in the list:
-        std::vector<std::wstring> f_unsup;
+        std::vector<CrowdinClient::FileInfo> f_unsup;
```

Comment:
All files are supported now. You removed all code adding anything to `f_unsup`. Why keep the variable?

Reply:
Addressed 085f4795857fc4df456ecf06c3f7da23d83c11bc . Sorry for just missing to remove initially

Reply:
Just if you @vslavik reviewed and approved it - please look once more since I have found and removed some more unsupported stuff related to the same peace of code in this commit